packages/client/src/pages/MainPage/MainPage.tsx

```diff
@@ -32,9 +38,22 @@ const MainPage: FC = () => {
         marginBottom={100}>
         <StMainLogo />
         <Button text="Начать игру" size="large" onClick={handlePlay} />
-        <Button css={marginTop20} text="Правила" disignType="alternate" />
+        <Button
+          onClick={handleOpenModal}
+          css={marginTop20}
+          text="Правила"
+          disignType="alternate"
+        />
       </StFlex>
+
+      <Modal
```

Comment:
Лучше не передавать isOpen в Modal, а прямо здесь делать проверку 
```suggestion
      { isOpen && <Modal ../>}
```